example/app.js

```diff
@@ -61,5 +61,5 @@ ds.discoverAndBuildModels(
         // ds.disconnect(); // This will crash node-oracle as the connection is disconnected while other statements are still running
       });
     });
-  }
+  },
```

Comment:
Can you please revert all unrelated changes where you are just adding a trailing comma to function args?

Reply:
Is it coming from the linter?

Reply:
I had to fix eslint errors to pass CI.